marconi-sidechain-experimental/test/Spec.hs

```diff
@@ -0,0 +1,22 @@
+{-# LANGUAGE ExplicitNamespaces #-}
+
+module Main (main) where
+
+import Marconi.ChainIndex.Logger (defaultStdOutLogger)
+import Network.JsonRpc.Client.Types ()
+import Spec.Marconi.Sidechain.Experimental.CLI qualified as CLI
+import System.Directory (getTemporaryDirectory)
+import System.IO.Temp (withTempDirectory)
+import Test.Tasty (TestTree, defaultMain, testGroup)
+
+-- Tests are run in a temporary directory for later use with Warp.testWithApplication,
+-- as tests analogous to those in marconi-sidechain are added.
+main :: IO ()
+main = do
+  tmp <- getTemporaryDirectory
+  withTempDirectory tmp "marconi-sidechain" $ \tempDir -> do
```

Comment:
In fact, these CLI golden tests do not use the temporary directory at all. I have a follow-up ticket open to create a better test structure for this package, since the tests for RPC stuff cannot be copied directly from the original `marconi-sidechain`.

So I removed the `withTempDirectory` entirely from the work here, to keep it clean as it is not needed.